src/components/board-column/column.tsx

```diff
@@ -0,0 +1,14 @@
+import { component$, useStylesScoped$ } from "@builder.io/qwik";
+import { Square } from "../board-square/square";
+import styles from "./column.css?inline";
+
+export const BoardColumn = component$(() => {
+	useStylesScoped$(styles);
+
+	const squares = [];
+	for (let i = 0; i < 5; i++) {
```

Comment:
[**A comment** (PR comments with a purpose)](https://gist.github.com/dgoings/ecfe4877b5992190e4fc8893c77a7ae6):
Uh oh, magic # `5`. This should get fixed once we have a state we're rendering from.